source/release_notes/3.0.5-rc7.rst

```diff
@@ -6,7 +6,11 @@ Percona Server for MongoDB 3.0.5-rc7
 
 Percona is glad to announce the release of Percona Server for MongoDB on September 16th, 2015. Downloads are available `here <https://www.percona.com/downloads/percona-server-for-mongodb>`_ and from the :ref:`Percona Software Repositories <installation>`.
 
+Percona Server for MongoDB is a highly scalable, zero-maintenance downtime database supporting the MongoDB v3.0 protocol and drivers. It extends MongoDB with RocksDB and PerconaFT storage engines, as well as features like external authentication and audit logging. Percona Server for MongoDB requires no changes to MongoDB applications or code.
```

Comment:
Which version of MongoDB (link to the release notes) was this release based on? It would be good to know which versions of RocksDB and PerconaFT are included so users are aware what version/features are they installing.